### ISSUE TITLE
performance bottleneck on CPU or Disk HDD/SSD

### ISSUE THREAD

**Issue by paulheisterkamp97** [AlexeyAB/darknet#6149]
Hi, 
I got a bottleneck problem and cant figure out why.
Im training on a GPU server of my university,  i have to use docker bc of permission handling. 

I copied this docker img https://hub.docker.com/r/alemelis/darknet and build it against my laptop (to get the newest version of darknet + yolov4). 
with this make line (just changed the thread count to 40):
RUN make OPENCV=1 GPU=1 AVX=1 OPENMP=1 CUDNN=1 CUDNN_HALF=1 OPENMP=1 -j 40

However I'm getting this kind of loading times for ca 2/3 of all batches:
Loaded: 1.174679 seconds - performance bottleneck on CPU or Disk HDD/SSD
The rest computes just fine like this: 
Loaded: 0.000078 seconds

If im using the original docker img there are no loading problems at all, using the same cfg file as before.

The GPU spikes at 70-90% but is not under constant load with memory usage between 10-22G.
CPU usage maxing at 50%

 Info on training start:
 CUDA-version: 10000 (10020), cuDNN: 7.6.5, CUDNN_HALF=1, GPU count: 1
 OpenCV version: 3.2.0
 CUDNN_HALF=1
 0 : compute_capability = 700, cudnn_half = 1, GPU: Tesla V100-SXM2-32GB

head of cfg file:
```javascript
[net]
subdivisions=8
width=608
height=608
channels=3
momentum=0.949
decay=0.0005
max_chart_loss=5

#augentation
angle=30
saturation = 1.5
exposure = 1.5
hue=.1
mosaic=1
mosaic_bound=1
min_crop=224
max_crop=448

learning_rate=0.001
burn_in=1000
max_batches = 30000
policy=steps
steps= 15000,25000
scales=.1,.1
```

**Comment by RainbowSun11Q2H** [comment]
The same issue and the loss is going to nan.
Do you solve this problem?
